lib/kredis/types/flag.rb

```diff
@@ -1,8 +1,8 @@
 class Kredis::Types::Flag < Kredis::Types::Proxying
   proxying :set, :exists?, :del
 
-  def mark(expires_in: nil)
-    set 1, ex: expires_in
+  def mark(expires_in: nil, unless_exists: false)
+    !!(set 1, ex: expires_in, nx: unless_exists)
```

Comment:
have to use `!!` because with `nx: false` it returns `OK` vs boolean with `nx: true`

Reply:
We don't guard against `"OK"` in any other client calls, we rely on it being truthy in Ruby, so let's just do:

```suggestion
    set 1, ex: expires_in, nx: unless_exists
```